### PR TITLE
SW-2106 Flow to add a planting site

### PR DESCRIPTION
**Pull request by constanzauanini**
Screenshot 2022-11-11 at 13 50 08

Screenshot 2022-11-11 at 13 47 39

### CODE REVIEW

File: src/components/PlantingSites/PlantingSiteCreate.tsx [hunk 1]
```diff
@@ -1,10 +1,150 @@
...
+              }}
+            >
+              <Typography fontSize='20px' fontWeight={600} margin={theme.spacing(3, 0)}>
+                {strings.IMPORT_BOUNDARIES_AND_PLOTS}
+              </Typography>
+              <Typography fontSize='16px' fontWeight={400} padding={theme.spacing(1, 0)}>
+                {strings.IMPORT_BOUNDARIES_AND_PLOTS_DESCRIPTION}
+              </Typography>
+              <Box sx={{ paddingY: 2 }}>
+                <Button label={strings.CONTACT_US} onClick={() => true} size='medium' />
```
Comment (karthikbtf) [inline src/components/PlantingSites/PlantingSiteCreate.tsx:141]: for now maybe we can do the same thing the contact us card does when we click?
i understand there's a jira out for allison on what this should do.

File: src/components/PlantingSites/index.tsx [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import CreatePlantingSite from './PlantingSitesList';
-import PlantingSitesList from './PlantingSiteCreate';
+import CreatePlantingSite from './PlantingSiteCreate';
+import PlantingSitesList from './PlantingSitesList';
```
Comment (karthikbtf) [inline src/components/PlantingSites/index.tsx:2]: woops :) good catch

File: src/components/PlantingSites/PlantingSiteCreate.tsx [hunk 3]
```diff
@@ -1,10 +1,150 @@
...
+            </Typography>
+            <Typography fontSize='14px' fontWeight={400} margin={theme.spacing(0, 0, 3, 0)}>
+              {strings.ADD_PLANTING_SITE_DESCRIPTION}
+            </Typography>
+          </Grid>
+          <PageSnackbar />
+          <Grid item xs={gridSize()}>
+            <TextField
+              id='name'
+              label={strings.NAME}
```
Comment (karthikbtf) [inline src/components/PlantingSites/PlantingSiteCreate.tsx:102]: I understand Name and Description don't have the required '*' next to them - but feels like they should? 
We could check with @chudo76 in figma.

File: src/components/PlantingSites/PlantingSiteCreate.tsx (hunk 1 above)
Comment (constanzauanini) [reply src/components/PlantingSites/PlantingSiteCreate.tsx:141]: Oh, yes, I can add that for now!

File: src/components/PlantingSites/PlantingSiteCreate.tsx (hunk 3 above)
Comment (constanzauanini) [reply src/components/PlantingSites/PlantingSiteCreate.tsx:102]: Name is required according to the api, and description is optional. I'm validating name when saving

Comment (karthikbtf) [reply src/components/PlantingSites/PlantingSiteCreate.tsx]: PRD says name alone is required - which is different from our other pages.
We should at least ask Chudo if we should add the '*' next to 'Name' label for consistency, yes?

**Review by karthikbtf — APPROVED** [review]
looks good, we can follow up on name label when chudo responds on figma.
thanks for asking the question there.

**Comment by constanzauanini** [comment]
> looks good, we can follow up on name label when chudo responds on figma. thanks for asking the question there.



> looks good, we can follow up on name label when chudo responds on figma. thanks for asking the question there.

Yes, I added the * for consistency, I can remove it later if we don't need it

**Pull request merged**